apps/web/src/components/widget/AvatarContainer.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { createStyles } from '@mantine/core';
+import styled from '@emotion/styled';
+import { colors } from '../../design-system';
+
+export const useStyles = createStyles((theme) => {
+  const dark = theme.colorScheme === 'dark';
+
+  return {
+    inner: { margin: 0, padding: 15 },
+    target: { height: '40px' },
+    arrow: {
+      backgroundColor: dark ? colors.B20 : colors.white,
+      marginBottom: '-1px',
```

Comment:
Are you using the negative margin to position the arrow icon?

Reply:
Actually, it's not needed after changing the placement,
will remove it, thanks